### DISCUSSION TITLE
Please add SRID 900913 for the spatial viewer

### DISCUSSION THREAD

**Discussion by djagar** [dbeaver/dbeaver#13430] [Q&A]
According to https://epsg.io/900913 it is equivalent to SRID/EPSG 3857, so an alias could suffice.
Alternatively please provide a hint, how custom SRIDs can be added.

**Comment by uslss** [comment]
You can read more about it [here](https://github.com/dbeaver/dbeaver/wiki/Working-with-Spatial-GIS-data#tile-layer-management)
